data/json/snippets/lab.json

```diff
@@ -266,122 +266,470 @@
+      },
+      {
+        "id": "labnote_64",
+        "text": "We contacted colleagues at XEDRA-40 via Melchior.  They ran their blood too, and got the same result we did.  They're initiating quarantine protocol as well, but now the word is out and Melchior is getting everyone to run a similar protocol.  Dr. Dionne thinks it's probably too late: we've got a full scale containment breach on our hands.  How did this happen?  We're backtracing, but we've been beyond paranoid with our protocols, it's not like we're dealing with butterscotch pudding here."
+      },
+      {
+        "id": "labnote_65",
+        "text": "A backtrace didn't find any evidence of a protocol breach.  We're getting reports from other XEDRA facilities now.  Contamination isn't 100%, but it's widespread.  The non-quarantined facilities are sending people out to check the surrounding areas for signs of XE037 escaping the lab.  We're running some experiments on XE037 to see how it could have broken containment."
+      },
+      {
+        "id": "labnote_66",
+        "text": "After we found the mutation in Hendelson's mouse, we checked a few of the others.  Most weren't so dramatic, but a few others had developed similar changes.  Size increase is the most common, but a couple others had strange new organs.  One had tentacles squirming under its skin."
+      },
+      {
+        "id": "labnote_67",
+        "text": "We exposed the XE037 colonized mice to an assortment of mutagenic and carcinogenic compounds today.  High dose radiation was interesting: too much still killed the mice, but a sub-fatal dose caused rapid mutation, incredibly rapid.  It turned yellow, grew to the size of a cat, and started shooting lightning bolts everywhere.  About a minute later it died, but that was a hell of a lightshow in the meantime."
+      },
+      {
+        "id": "labnote_68",
+        "text": "Today we applied a very small sample of XE037, suspended in water, to subject TP92, prior to necessary termination.  During the postmortem examination, the subject went through a revivification progress.  Dr. Thoreaux was surprised, to say the least.  She bludgeoned it back to death with a microscope."
+      },
+      {
+        "id": "labnote_69",
+        "text": "We have created a new department to investigate the effects of XE037 on the human body, specifically to examine the revivification effect observed on two prior occasions.  Sadly, our human subject pool is dwindling due to the short lifespan of subjects in this program.  More will be acquired soon."
+      },
+      {
+        "id": "labnote_70",
+        "text": "An experimental subject was accidentally terminated today, and revivification was again seen as in Dr. Thoreaux's report.  This time we were ready for it, and the specimen was kept for observation.  It displays no human intelligence, a true animated corpse.  We're all calling it a zombie, because seriously.  This is so cool."
```

Comment:
```suggestion
        "effect_on_examine": [ { "u_add_var": "u_has_read_labnote_70", "type": "dialogue", "context": "world_knowledge", "value": "yes" } ],
        "text": "An experimental subject was accidentally terminated today, and revivification was again seen as in Dr. Thoreaux's report.  This time we were ready for it, and the specimen was kept for observation.  It displays no human intelligence, a true animated corpse.  We're all calling it a zombie, because seriously.  This is so cool."
```

Reply:
Oh, how annoying, it looks like a number of my commits didn't take and are still on my work computer.